### DISCUSSION TITLE
tabby + zsh

### DISCUSSION THREAD

**Discussion by trotroyanas** [Eugeny/tabby#5279] [Q&A]
hi i have a encodage problem ?
you have a idea ?
on terminator bottom is ok
![terminator](https://user-images.githubusercontent.com/4205983/147286459-4e19ebb7-40d1-460a-856a-b0b7bbfe8563.png)

on tabby error with fonts ?
![tabby](https://user-images.githubusercontent.com/4205983/147286503-fcae2342-d65b-4c5f-b036-e4a570e33728.png)

**Comment by Eugeny** [comment]
You should use the same font you're using in Terminator. These squares mean missing characters - your current font in Tabby doesn't include the Powerline icons.

**Comment by trotroyanas** [comment]
i found "Source Code ProdRegular 10" in terminator
but i no this font in tabby ???

**Reply by cpecorari** [reply]
There are issues with some fonts not showing up in tabby settings, but you can force font name in settings confg file at the bottom, look for 
```sh
terminal: [..]
  font: MesloLGS NF #or any font name you want here
```
it should be ok that way !

**Reply by Eugeny** [reply]
If your font doesn't show up in the font selector, could you please post a link to the font file so that I can try it out? As @cpecorari said you can always set the font name explicitly (you can also just type it into the font selector field even if it doesn't auto-complete).

**Comment by trotroyanas** [comment]
![image](https://user-images.githubusercontent.com/4205983/147849532-cff44b41-83e1-41fe-9b82-adfd70594f12.png)
nothing :(
i don't understand, everything looks good

**Reply by ChJus** [reply]
I'm guessing you might not have Source Code Pro (Regular) installed on your computer (perhaps because Terminator directly included the font file externally)? You can get it from [Google Fonts](https://fonts.google.com/specimen/Source+Code+Pro?query=source+code+pro).
Also, the font name probably should be `Source Code Pro` instead of `Source Code Pro Regular`.

**Comment by trotroyanas** [comment]
![image](https://user-images.githubusercontent.com/4205983/147880567-e1c95efc-03d8-46d9-8108-d362a796a6ea.png)
i have change Source Code Pro
i think font are installed correctly ?

**Comment by braniubojni** [comment]
Hi, I have the same problem. I've installed [Hack Nerd Font](https://www.nerdfonts.com/font-downloads) and icons started to show
![image](https://user-images.githubusercontent.com/29413886/158513344-b525a84b-4709-47db-a697-1a583963ec94.png)
Here example
![image](https://user-images.githubusercontent.com/29413886/158514843-40de2949-4759-4bef-b5c7-74cc07ef8eee.png)

**Reply by trotroyanas** [reply]
thank you so much.
i work fine now :)

**Comment by nothoda** [comment]
> hi i have a encodage problem ? you have a idea ? on terminator bottom is ok ![terminator](https://user-images.githubusercontent.com/4205983/147286459-4e19ebb7-40d1-460a-856a-b0b7bbfe8563.png)
> 
> on tabby error with fonts ? ![tabby](https://user-images.githubusercontent.com/4205983/147286503-fcae2342-d65b-4c5f-b036-e4a570e33728.png)

**Reply by braniubojni** [reply]
- Install Nerd fonts
- restart terminal
- find in terminal fonts and pick one of nerd fonts that you installed
Should work now
